packages/next/client/index.js

```diff
@@ -182,17 +182,6 @@ export default async ({ webpackHMR: passedWebpackHMR } = {}) => {
   const renderCtx = { App, Component, props, err: initialErr, emitter }
   render(renderCtx)
 
-  if (window.__NEXT_DATA__.skeleton && Component.getInitialProps) {
```

Comment:
Since this code is gone what causes the `getInitialProps` invoke when the skeleton mounts? Is this handled by the `router.replace` method above?

Reply:
> Is this handled by the router.replace method above

Yeah it's handled there